docs/ocs/troubleshooting.md

```diff
@@ -0,0 +1,169 @@
+  - From outside the cluster?
+- If you create a new `ObjectBucketClaim`, does it bind immediately to
+  a new bucket?
+
+If *some* of the above steps work, there is (probably) a software
+problem on the cluster. Continue with "Troubleshooting software
+problems".
+
+If *none* of the above steps work, it may be a connectivity problem
+with the Ceph service; continue with "Troubleshooting Ceph
+connectivity".
+
+## Troubleshooting software problems
+
+Generally, troubleshooting problems with the storage subsystem is
+going to be very much like troubleshooting issues with any other
+application running on OpenShift:
+
+- Look for unhealthy pods (`oc get pod | grep -Evi
+  'running|completed'`).
+
+- Inspect the logs of unhealthy pods (`oc logs ...`).
+
+- Inspect events for any errors (`oc get events`).
+
+### Noobaa object storage
+
+To check the status of the Noobaa object storage service, install the
+Noobaa CLI from the [releases
+page](https://github.com/noobaa/noobaa-operator/releases) and then
+run:
+
+```
+noobaa -n openshift-storage status
+```
+
+This will emit a variety of validation messages on *stderr* and some
+status information on *stdout*.  The validation messages look
+something like:
+
+```
+time="2021-11-23T17:07:41-05:00" level=info msg="CRD Status:"
+time="2021-11-23T17:07:41-05:00" level=info msg="✅ Exists: CustomResourceDefinition \"noobaas.noobaa.io\"\n"
+[...]
+time="2021-11-23T17:07:41-05:00" level=info msg="Operator Status:"
+time="2021-11-23T17:07:41-05:00" level=info msg="✅ Exists: Namespace \"openshift-storage\"\n"
+time="2021-11-23T17:07:41-05:00" level=info msg="✅ Exists: ServiceAccount \"noobaa\"\n"
+time="2021-11-23T17:07:41-05:00" level=info msg="✅ Exists: ServiceAccount \"noobaa-endpoint\"\n"
+[...]
+time="2021-11-23T17:07:41-05:00" level=info msg="System Wait Ready:"
+time="2021-11-23T17:07:41-05:00" level=info msg="✅ System Phase is \"Ready\".\n"
+time="2021-11-23T17:07:41-05:00" level=info
+time="2021-11-23T17:07:41-05:00" level=info
+time="2021-11-23T17:07:41-05:00" level=info msg="System Status:"
+time="2021-11-23T17:07:41-05:00" level=info msg="✅ Exists: NooBaa \"noobaa\"\n"
+time="2021-11-23T17:07:41-05:00" level=info msg="✅ Exists: StatefulSet \"noobaa-core\"\n"
+[...]
+time="2021-11-23T17:07:42-05:00" level=info msg="✅ System Phase is \"Ready\"\n"
+time="2021-11-23T17:07:42-05:00" level=info msg="✅ Exists:  \"noobaa-admin\"\n"
+```
+
+The status information includes various configuration details,
+including the external URL for the Noobaa management interface. You
+should be able to log into that UI using your OpenShift credentials.
+
+## Troubleshooting Ceph connectivity
+
+You can log into the `rook-ceph-tools` container to run Ceph commands
+to diagnose Ceph cluster health and connectivity problems. To log into
+the pod:
+
+```
+oc -n openshift-storage rsh deploy/rook-ceph-tools
+```
+
+If this deployment isn't available, you can enable it by [editing the
+`OCSInitialization`
```

Comment:
can we mention that to access this you need a red hat account? (if there is another non gated link, even better)

Reply:
I don't think there's any non-gated link for this.